docs/how-payjoin-saves-exchanges.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 1
+---
+
+# How Can Payjoin Save 1/3 the Cost of Transacting?
+
+Payment batching is table stakes service for high-volume settlement services like exchanges and payment processors. But it has been limited to one party, the sender, combining multiple sends together. Ideally, multiple types of transfers could all be combined together. Imagine your deposit to an exchange was batched with withdrawals to others and lightning channel opens etc. This combination could save significant overhead to every transfer. And it provides an opportunity for much better privacy since, like bitcoin itself, only inputs and outputs are recorded but less information about which inputs and outputs are clustered would be revealed.
```

Comment:
"table stakes service"?

Reply:
How about "Payment batching is a necessary, common optimization high-volume settlement services like exchanges and payment processors."